__tests__/nc_news_db.test.js

```diff
@@ -52,3 +51,32 @@ describe("GET api/topics", () => {
 			});
 	});
 });
+
+describe("GET /api/articles/:articles_id", () => {
+	it("status: 200, should respond with the requested article", () => {
+		return request(app)
+			.get("/api/articles/1")
+			.expect(200)
+			.then(({ body }) => {
+				expect(body).toEqual({
+					article_id: expect.any(Number),
+					title: expect.any(String),
+					topic: expect.any(String),
+					author: expect.any(String),
+					body: expect.any(String),
+					created_at: expect.any(String),
+					votes: expect.any(Number),
+					article_img_url: expect.any(String),
+				});
+			});
+	});
+
+	it("status: 404, should respond with a 404 message when passed a valid but none existent article ID", () => {
+		return request(app)
+			.get("/api/articles/808")
+			.expect(404)
+			.then(({ body }) => {
+				expect(body.message).toBe("no article found under article_id 808");
+			});
+	});
+});
```

Comment:
Your missing a test for a: **Status 400, invalid ID, e.g. string of "not-an-id"**